zero/gkyl_dg_rad_gyrokinetic_drag.h

```diff
@@ -12,6 +12,8 @@ struct gkyl_dg_rad_gyrokinetic_auxfields {
   const struct gkyl_array *nvnu;
   const struct gkyl_array *nvsqnu_surf;
   const struct gkyl_array *nvsqnu;
+  const struct gkyl_array *vtsq;
+  double vtsq_min;
```

Comment:
@jRoeltgen @akashukla 
I suggest you add comments describing what each of these aux fields is, especially since this is a public header so it's supposed to be very clear.